docs/README.md

```diff
@@ -76,16 +77,19 @@ For example:
     "markers": [
       "highlight"
     ],
-    "blockClass": "summary"
+    "blockClass": "summary",
+    "message": "{sellingPriceValue}"
```

Comment:
```suggestion
    "message": "Selling price"
```

Reply:
> Thank you so much, Hugo! I was about to update this when the notification came up ❤️

You are welcome, sorry about my poor writing.

About your suggestion ("message": "Selling price"), I think that will be a good choice to put the variable itself at the example, to really show the correct usage, I struggle a bit trying to make this work kkkkkk.

Reply:
No needs to apologize!!! It was a great contribution 💯 

Since this is a practical example I don't see why we need to use the variable itself. In practice, users will use the `message` prop to declare which message text they want to be rendered, right?

Reply:
Correct, but I think it will be great to show the possibility to use the variables because the product-price blocks are heavily dependent by them.

Reply:
Why do you think we should use the variable as the `message` prop value? I mean, the product price blocks are heavily dependent on the variables because of the `markers` prop. I don't see the link between `message`  prop and the variables.

Reply:
I don't know the motive behind this feature.  I used that way, I edit the message showed with the variables, because not always the clients wanted the default message displayed. E.g, If I want to add "OFF" string after {sellingPriceValue} I just use this: "message": " {sellingPriceValue} OFF". In that way, I'm free to customize the messages of product-price as I want.

Reply:
I believe @iaronaraujo can confirm this!

Reply:
I think it is better to merge them and use the example 
"message": "Selling price: {sellingPriceValue}"
as @hugocostadev mentioned!
@guerreirobeatriz in this case it is important to show the variable so the people using this will know that you can use variables on this prop the same way you can on site editor!

Reply:
I will update the changes. Thanks.